src/charity_base.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# Taken from their website's "What is CharityBase?"
+description: >
+  Charity Base is an API that aggregates data from the 168,000 charities
+  registered in England and Wales.
+# One Project is also licensed under Apache-2.0 and some don't have
+# a license
+license: MIT
+name: CharityBase
+programmingLanguages:
+  - javascript
+  - python
+rating: 2 # Because of the bad documentation and Contribution guide
```

Comment:
How about writing the comment as 
```yaml
rating : 2
ratingComment: The documentation and contribution guide could be improved
```

This way, we can show the comment as an help text or info bubble on the website, and within a details tag for example on markdown

Reply:
Oh, I like the idea :)
Originally, I thought, I could summarize the rules for the rating it received in a bubble or similar, but I also like custom comments.
I'd say, we add them as optional fields?
If so, I'd probably not do it in this PR, but instead create one, where I add it to the template, documentation and go through the currently listed projects to add one where appropriate.